packages/documentsearch/src/searchview.tsx

```diff
@@ -265,19 +266,41 @@ function ReplaceEntry(props: IReplaceEntryProps): JSX.Element {
 }
 
 interface IUpDownProps {
+  commands: CommandRegistry;
   onHighlightPrevious: () => void;
   onHighlightNext: () => void;
   trans: TranslationBundle;
 }
 
 function UpDownButtons(props: IUpDownProps) {
+  // Getting the key bindings for the next and previous commands
+  const nextBinding = props.commands.keyBindings.find(
+    (binding: { command: string }) =>
+      binding.command === 'documentsearch:highlightNext'
+  );
+
+  const prevBinding = props.commands.keyBindings.find(
+    (binding: { command: string }) =>
+      binding.command === 'documentsearch:highlightPrevious'
+  );
+
+  const nextKeys = nextBinding
+    ? CommandRegistry.formatKeystroke(nextBinding.keys)
+    : '';
+  const prevKeys = prevBinding
+    ? CommandRegistry.formatKeystroke(prevBinding.keys)
+    : '';
+
+  const prevShortcut = prevKeys ? `(${prevKeys})` : '';
+  const nextShortcut = nextKeys ? `(${nextKeys})` : '';
+
   return (
     <div className={UP_DOWN_BUTTON_WRAPPER_CLASS}>
       <button
         className={BUTTON_WRAPPER_CLASS}
         onClick={() => props.onHighlightPrevious()}
         tabIndex={0}
-        title={props.trans.__('Previous Match')}
+        title={`${props.trans.__('Previous Match')} ${prevShortcut}`}
```

Comment:
Unneeded as per previous suggestion

```suggestion
        title={`${props.trans.__('Previous Match')}${prevShortcut}`}
```